### PR TITLE
Add fastertramsformer default handler

### PR DESCRIPTION
**Pull request by rohithkrn**
## Description ##

Add fastertramsformer default handler

Tested locally by passing as model.py

### CODE REVIEW

File: engines/python/setup/djl_python/fastertransformer.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        self.model_id = properties.get("model_id")
+        self.model = self.load_model()
+        self.initialized = True
+
+    def load_model(self):
+        logging.info(f"Loading model: {self.model_id}")
+        return ft.init_inference(self.model_id, self.tensor_parallel_degree,
+                                    self.pipeline_parallel_degree, self.dtype)
+
+    def inference(self, inputs: Input):
```
Comment (lanking520) [inline engines/python/setup/djl_python/fastertransformer.py:23]: add a parameter field to allow customer do customized setup on beam search and other settings

Comment (lanking520) [reply engines/python/setup/djl_python/fastertransformer.py:23]: https://github.com/deepjavalibrary/djl-serving/blob/master/engines/python/setup/djl_python/huggingface.py#L127

File: engines/python/setup/djl_python/fastertransformer.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
+import fastertransformer as ft
```
Comment (frankfliu) [inline engines/python/setup/djl_python/fastertransformer.py:1]: 1. Add license header
2. run: `./gradlew formatPython`

File: engines/python/setup/djl_python/fastertransformer.py [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+class FasterTransformerService(object):
+    def __init__(self) -> None:
+        self.initialized = False
+
+    def initialize(self, properties):
+        self.tensor_parallel_degree = int(properties.get("tensor_parallel_degree", 1))
+        self.pipeline_parallel_degree = int(properties.get("pipeline_parallel_degree", 1))
+        self.dtype = properties.get("dtype", "fp32")
+        self.model_id = properties.get("model_id")
```
Comment (sindhuvahinis) [inline engines/python/setup/djl_python/fastertransformer.py:14]: model_dir can be considered as well. Like this [https://github.com/deepjavalibrary/djl-serving/blob/master/engines/python/setup/djl_python/huggingface.py#L69](https://github.com/deepjavalibrary/djl-serving/blob/master/engines/python/setup/djl_python/huggingface.py#L69)

Comment (rohithkrn) [reply engines/python/setup/djl_python/fastertransformer.py:14]: right now, model_dir is not part of FT yet. Will add that support in the future.

File: engines/python/setup/djl_python/fastertransformer.py (hunk 2 above)
Comment (rohithkrn) [reply engines/python/setup/djl_python/fastertransformer.py:14]: done

File: engines/python/setup/djl_python/fastertransformer.py (hunk 1 above)
Comment (rohithkrn) [reply engines/python/setup/djl_python/fastertransformer.py:39]: done

**Review by lanking520 — APPROVED** [review]

File: engines/python/setup/djl_python/fastertransformer.py [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+    def __init__(self) -> None:
+        self.initialized = False
+
+    def initialize(self, properties):
+        self.tensor_parallel_degree = int(
+            properties.get("tensor_parallel_degree", 1))
+        self.pipeline_parallel_degree = int(
+            properties.get("pipeline_parallel_degree", 1))
+        self.dtype = properties.get("dtype", "fp32")
+        self.model_id = properties.get("model_id")
```
Comment (lanking520) [inline engines/python/setup/djl_python/fastertransformer.py:30]: Please also add model dir solution here. User may still want to have the full model in a tarball (JumpStart)

Comment (rohithkrn) [reply engines/python/setup/djl_python/fastertransformer.py:30]: Yes, will add after some testing.

**Pull request merged**